lib/next_ls/commands/from_pipe.ex

```diff
@@ -0,0 +1,116 @@
+defmodule NextLS.Commands.FromPipe do
+  @moduledoc false
+  alias GenLSP.Enumerations.ErrorCodes
+  alias GenLSP.Structures.Position
+  alias GenLSP.Structures.Range
+  alias GenLSP.Structures.TextEdit
+  alias GenLSP.Structures.WorkspaceEdit
+
+  defp opts do
+    Schematic.map(%{
+      position: Schematic.map(%{character: Schematic.int(), line: Schematic.int()}),
```

Comment:
For this, you can actually do `GenLSP.Structures.Position.schematic()` (i think)